src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/Subscriptions/DependenciesSnapshotProvider.cs

```diff
@@ -311,7 +311,8 @@ private void DisposeCore()
         private Task OnConfiguredProjectEvaluatedAsync(IProjectVersionedValue<IProjectSubscriptionUpdate> e)
         {
             // If "TargetFrameworks" property has changed, we need to refresh the project context and subscriptions.
-            if (HasTargetFrameworksChanged())
+            // If no context exists yet, create one.
+            if (HasTargetFrameworksChanged() || _context.Current == null)
```

Comment:
does the old behavior result in a yellow bar warning?

Reply:
No, the old behaviour fails silently. It would never create the necessary data flow subscriptions, so the Dependencies node would appear empty indefinitely.